components/App.js

```diff
@@ -0,0 +1,82 @@
+var GIPHY_API_URL = "http://api.giphy.com";
+var GIPHY_PUB_KEY = "dc6zaTOxFJmzC";
+function httpGet(url_in) {
```

Comment:
wrzuc to jako metode do APP -
tak jak masz getInitialState() {}
Tak zrob 
httpGet(url) {KOD}

reszta jest ok